### PR TITLE
DOCSP-18445 results sections

### PR DESCRIPTION
**Pull request by kyuan-mongodb**
## Pull Request Info

 Updated all results sections to the following:
- The wording with the format:  `"After you run the preceding example, ..."`
- Link to the find pages to show how you can view the result of what was just done (all CRUD pages)

### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-18445

### Snooty build log:
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?jobId=6137e2ae7dcf0129309016b0

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/golang/docsworker-xlarge/DOCSP-18445-ResultsSection/usage-examples/

### Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Does it render on staging correctly?
- [x] Are all the links working?
- [x] Are the staging and workerpool job links in the PR description updated?

### If your page documents a concept, does it meet the following criteria?

- [ ] Target the [Jasmin persona](https://drive.google.com/file/d/14FbBOLCVxwSP6M9BK4Nz1Ir9tzxT8_02/view)
- [ ] Target the [Lucas persona](https://drive.google.com/file/d/1J2vqJxo7ldv7OP_obA9Q-avf0o_ju4Lk/view)

### CODE REVIEW

File: source/usage-examples/insertMany.txt [hunk 1]
```diff
@@ -30,7 +30,7 @@ Click `here <{+example+}/insertMany.go>`__ to see a fully runnable example.
 Expected Result
 ---------------
 
-After you run the code example, two new documents will be present in the
+After you run the preceding example, two new documents will be present in the
```
Comment (ccho-mongodb) [inline source/usage-examples/insertMany.txt:33]: This mixes present tense ("After you run the preceding example") and continuous future tense ("documents will be present") in a sentence which makes it less clear.
The style guide suggests only using present tense: https://docs.mongodb.com/meta/style-guide/writing/use-present-tense/

A couple ways you can reword the second phrase which seem to match up with how the other usage examples are worded:
- "your collection should contain two new documents..."
- "you can find two new documents..."

Applies to all instances of future tense.

**Review by ccho-mongodb — CHANGES_REQUESTED** [review]
Looks mostly good. However, we should avoid future tense per the style guide and for clarity. Please update these to use present tense.

File: source/usage-examples/distinct.txt [hunk 2]
```diff
@@ -26,8 +26,8 @@ Click `here <{+example+}/distinct.go>`__ to see a fully runnable example.
 Expected Result
 ---------------
 
-The example returns an empty slice of an ``interface`` type that
-contains the following values:
+After you run the preceding example, it returns an empty slice of an
+``interface`` type that contains the following values:
```
Comment (ccho-mongodb) [inline source/usage-examples/distinct.txt:30]: Question: I'm confused what is returned. If it's an empty slice, why does it contain values? Or are you trying to say that the interface contains values? I would reword this to clarify this ambiguity.

File: source/usage-examples/find.txt [hunk 3]
```diff
@@ -25,8 +25,8 @@ Click `here <{+example+}/find.go>`__ to see a fully runnable example.
 Expected Result
 ---------------
 
-This example creates a ``Cursor`` object that returns
-documents similar to these:
+After you run the preceding example, it returns a ``Cursor`` object that
+contains the following documents:
```
Comment (ccho-mongodb) [inline source/usage-examples/find.txt:29]: I think the original text was correct in  saying that the Cursor returns documents. It does not contain documents.

File: source/usage-examples/findOne.txt [hunk 4]
```diff
@@ -25,7 +25,8 @@ Click `here <{+example+}/findOne.go>`__ to see a fully runnable example.
 Expected Result
 ---------------
 
-After you run the code example, you should see output similar to:
+After you run the preceding example, it returns a ``SingleResult``
+object with the following document:
```
Comment (ccho-mongodb) [inline source/usage-examples/findOne.txt:29]: I think "with" is ambiguous. If SingleResult contains documents, we should specify that. Otherwise, it could be interpreted as "alongside" meaning both a SingleResult object and the following documents are returned (in some unspecified object/format).

File: source/usage-examples/insertMany.txt [hunk 5]
```diff
@@ -30,8 +30,8 @@ Click `here <{+example+}/insertMany.go>`__ to see a fully runnable example.
 Expected Result
 ---------------
 
-After you run the code example, two new documents will be present in the
-collection with an ``_id`` field that matches the previous output:
+After you run the preceding example, you can find the inserted documents
```
Comment (ccho-mongodb) [inline source/usage-examples/insertMany.txt:33]: Nitpick: 

Seems like "following documents" is used in other places, but not here.

```suggestion
After you run the preceding example, you can find the following inserted documents
```

Applies to a few instances.

File: source/usage-examples/deleteMany.txt [hunk 6]
```diff
@@ -26,8 +26,8 @@ Click here <TODO> to see a fully runnable example.
 Expected Result
 ---------------
 
-After running the preceding code snippet, you should not be able to find
-the following documents in the ``movies`` collection:
+After you run the preceding example, you cannot find the following
+documents in the ``movies`` collection:
```
[the remark this answers is not in the document]
Comment (ccho-mongodb) [reply source/usage-examples/deleteMany.txt:30]: Nitpick:
I wonder if we need to mention the collection or if that's info we can expect them to deduce if we're not referencing any other collection in the example. I'm curious to hear your thoughts though.

**Review by ccho-mongodb — APPROVED** [review]
LGTM. I added a couple style guide suggestions, nitpicks, and questions. Could you respond to them (or address anything that seems reasonable) before merging? Thanks!

File: source/usage-examples/count.txt [hunk 7]
```diff
@@ -29,7 +29,7 @@ Click `here <{+example+}/count.go>`__ to see a fully runnable example.
 Expected Result
 ---------------
 
-After running the preceding code snippet, you should see the following:
+After you run the preceding example, you should see the following:
 
 - There are about ``23541`` documents in the ``movies`` collection
 - There are ``303`` documents in the ``movies`` collection from "China"
```
[the remark this answers is not in the document]
Comment (kyuan-mongodb) [reply source/usage-examples/count.txt:35]: Agreed, I will change the ending to "... that contain "China" in the ``countries`` field"

File: source/usage-examples/deleteMany.txt (hunk 6 above)
[the remark this answers is not in the document]
Comment (kyuan-mongodb) [reply source/usage-examples/deleteMany.txt:30]: Agreed, will change "you cannot find" to "it removes"

As for mentioning the collection, I think it should be included. Although there is only one collection on each page, the sentence seems incomplete without saying where the result we're showing is from.

File: source/usage-examples/find.txt (hunk 3 above)
Comment (kyuan-mongodb) [reply source/usage-examples/find.txt:29]: Will change "that contains" to "with"

Comment (kyuan-mongodb) [reply source/usage-examples/find.txt]: I'll change this to say "...  it creates a ``Cursor`` object that returns the following documents:"

File: source/usage-examples/distinct.txt (hunk 2 above)
Comment (kyuan-mongodb) [reply source/usage-examples/distinct.txt:30]: So in Go, an [empty interface](https://tour.golang.org/methods/14) is a term used to describe `interface{}` regardless of how many documents it contains.
